### DISCUSSION TITLE
The location of the settings.json file

### DISCUSSION THREAD

**Discussion by alonaddy** [gitpod-io/openvscode-server#165] [Q&A] [answered]
Hi,

Where can I get access to the settings.json file (not from within vscode).

I would like to update the settings.json using a shell script ...

**Comment by asabi** [comment]
No one can answer this question? :-(

**Comment by akosyakov** [comment, marked as answer]
See https://github.com/gitpod-io/openvscode-server/discussions/199#discussioncomment-1568487

**Comment by hello-zhanghao** [comment]
if you want user the same settings, you can edit the settings.json in ~/.vscode-remote/data/Machine/settings.json.
Notice: the settings is the global setting, and when and where you connnect your openvscode, it is the same.
For example, it is the remote settings.
![image](https://user-images.githubusercontent.com/38358269/146141037-8f6b4914-6fd2-4599-a601-4856efa39bce.png)
![image](https://user-images.githubusercontent.com/38358269/146141135-3fed3b8f-ae1b-47c7-a6ca-d24e3b4f2641.png)

**Reply by sullemanhossam** [reply]
I tried to nano the location and navigate to it and nothing seems to be there
